content/en/blog/posts/2022-09-09-build-release-v0.11.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Shipwright v0.11.0 Is Here"
+date: 2022-09-09T12:00:01-04:00
+draft: false
+author: "Sascha Schwarze ([@SaschaSchwarze0](https://github.com/SaschaSchwarze0))"
+---
+
+Summer is over, and Shipwright is back with the v0.11.0 release.
+
+v0.11 is mostly a maintenance release without new features:
+
+* We updated dependencies to mitigate security vulnerabilities.
+* We updated our images to be based on the new Red Hat Universal Base Image 9 Minimal.
+* We updated build strategy tools as usual. This includes a fix that re-enabled the ko build strategy. Congratulations to them for moving into the own organization. It is a great build tool that we do not just provide a sample for, but also use in our own builds.
+
+Behind the scenes we are working on streamlining our API objects. As a first step, we deprecate the following fields:
+
+* In Builds, `spec.sources` is deprecated. We will consolidate supporting a single source for a Build under `spec.source`. Also, HTTP sources are deprecated because we think that the focus that secure software supply chain brings, does not match to artifacts that are loaded from an HTTP endpoint.
```

Comment:
Not 100% sure, but I think there is a slightly different notion between `to` and `with` here.
```suggestion
* In Builds, `spec.sources` is deprecated. We will consolidate supporting a single source for a Build under `spec.source`. Also, HTTP sources are deprecated because we think that the focus that secure software supply chain brings, does not match with artifacts that are loaded from an HTTP endpoint.
```